compass/LayersCalculator.java

```diff
@@ -77,8 +77,11 @@ public static void main(String[] args) throws IOException {
   public void run() {
     Path layersPath = Paths.get(config.layersPath);
     try {
-      Files.createDirectory(layersPath);
+      if (Files.notExists(layersPath)) {
+        Files.createDirectory(layersPath);
+      }
     } catch (IOException e) {
+      log.warn("failed to create folder: " + layersPath, e);
```

Comment:
Isn't this a fatal exception?
I mean how can you create the layers after this exception? Yet it is being caught...

Since this is a refactoring PR it can still be accepted and merged. I just want a reply before to understand that I am not missing anything

Reply:
I didn't want to break existing func.
I guess it was an empty `catch` b/c of the if exists.
I agree that it should probably throw - may open an issue?

Reply:
I'll open
You are on vacation